packages/puppeteer/README.md

```diff
@@ -8,9 +8,9 @@ Previous versions of this program were maintained at [dequelabs/axe-puppeteer](h
 
 Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already.
 
-Install Puppeteer: `npm install puppeteer --no-save`
```

Comment:
Not sure why we are promoting `--no-save` option here when it's a required dependency to run